### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by or-else**
* Fix for a crash in ChatListViewController
* Removed FindRouter because it's unused
* Fix for (sometimes) failing transitions from FindView to MessageView. Still not great because it takes ~200-300 ms to transition.
* Assigned MessageVC to rootController on basic login too

### CODE REVIEW

File: Tinodios/FindInteractor.swift [hunk 1]
```diff
@@ -30,7 +30,7 @@ class FindInteractor: FindBusinessLogic {
 
     static let kTinodeImProtocol = "Tinode"
     var presenter: FindPresentationLogic?
-    var router: FindRoutingLogic?
+    // var router: FindRoutingLogic?
```
Comment (aforge) [inline Tinodios/FindInteractor.swift]: Remove?

File: Tinodios/LoginViewController.swift [hunk 2]
```diff
@@ -59,7 +59,12 @@ class LoginViewController: UIViewController {
             let storyboard = UIStoryboard(name: "Main", bundle: nil)
             let destinationVC = storyboard.instantiateViewController(withIdentifier: "ChatsNavigator") as! UINavigationController
 
-            self.show(destinationVC, sender: nil)
+            if let window = UIApplication.shared.keyWindow {
+                // Use ChatList as the new root view controller (used in navigation).
+                window.rootViewController = destinationVC
+             }
+
+            // self.show(destinationVC, sender: nil)
```
Comment (aforge) [inline Tinodios/LoginViewController.swift]: Remove this.

File: Tinodios/LoginViewController.swift [hunk 3]
```diff
@@ -59,7 +59,12 @@ class LoginViewController: UIViewController {
             let storyboard = UIStoryboard(name: "Main", bundle: nil)
             let destinationVC = storyboard.instantiateViewController(withIdentifier: "ChatsNavigator") as! UINavigationController
 
-            self.show(destinationVC, sender: nil)
+            if let window = UIApplication.shared.keyWindow {
```
Comment (aforge) [inline Tinodios/LoginViewController.swift]: Interesting!
It turns out it's the recommended way to get |window| for Swift 4.2.
Can you also update UiUtils.routeToLoginVC to also follow this convention please?

Comment (or-else) [reply Tinodios/LoginViewController.swift]: done

File: Tinodios/LoginViewController.swift (hunk 2 above)
Comment (or-else) [reply Tinodios/LoginViewController.swift]: done

File: Tinodios/FindInteractor.swift (hunk 1 above)
Comment (or-else) [reply Tinodios/FindInteractor.swift]: done

**Pull request merged**